packages/core/src/client.ts

```diff
@@ -309,6 +309,18 @@ export class Client {
     return response$;
   };
 
+  subscription<Data = any, Variables extends object = {}>(
+    query: DocumentNode | string,
+    variables?: Variables,
+    context?: Partial<OperationContext>
+  ): Source<OperationResult<Data>> {
+    if (!context || typeof context.suspense !== 'boolean') {
+      context = { ...context, suspense: false };
+    }
+
```

Comment:
```suggestion
```

This isn't necessary as `suspense` is not applicable to subscriptions anyway

Reply:
I was thinking about that but wasn't sure as I don't have much experience with it yet.